src/patternfly/components/Drawer/drawer.scss

```diff
@@ -20,8 +19,9 @@ $pf-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
   --pf-c-drawer__panel--xl--FlexBasis: #{pf-size-prem(450px)};
   --pf-c-drawer__panel--ZIndex: var(--pf-global--ZIndex--sm);
   --pf-c-drawer__panel--BackgroundColor: var(--pf-global--BackgroundColor--100);
-  --pf-c-drawer__panel--TransitionDuration: .25s;
+  --pf-c-drawer__panel--TransitionDuration: var(--pf-global--TransitionDuration);
   --pf-c-drawer__panel--TransitionProperty: margin, transform, box-shadow;
+  --pf-c-drawer__panel--AnimationDelay: var(--pf-global--TransitionDuration);
```

Comment:
Is there a reason you need this var over just passing `--pf-c-drawer__panel--TransitionDuration` to the mixin?